rediscluster/pipeline.py

```diff
@@ -162,6 +163,10 @@ def send_cluster_commands(self, stack, raise_on_error=True, allow_redirections=T
 
             # now that we know the name of the node ( it's just a string in the form of host:port )
             # we can build a list of commands for each node.
+
+            if c.args[0] in ['MULTI', 'UNWATCH']:
```

Comment:
The comments that is above this belongs to the line below `node_name = node['name']`. Move the comments down or move this new if case up above the comments.